tests/util/utils.ts

```diff
@@ -0,0 +1,9 @@
+import { number } from "starknet";
+import BN__default from "bn.js";
+
+export type BigNumberish = string | number | BN__default;
```

Comment:
isn't this type already defined in starknet-js?

Reply:
can't access it :(